public-ui/src/app/data-browser/views/genomic-view/genomic-view.component.tsx

```diff
@@ -470,109 +467,127 @@ export const GenomicViewComponent = withRouteData(
+        participantCount, chartData, rowCount, searchTerm, filterMetadata } = this.state;
+      return <React.Fragment>
+        <style>{css}</style>
+        <div style={styles.pageHeader}>
+          <div style={styles.titleContainer}>
+            <h1 style={styles.title}>{this.title}</h1>
+          </div>
+          <div style={styles.viewLayout}>
+            <div style={styles.topBarLayout} id='topBar'>
+              {this.topBarItems.map((item, index) => {
+                return <div key={index} style={styles.topBarItemContainer}>
+                  <div onClick={() => this.topBarClick(item.id)}
+                    style={{ ...selectionId === item.id && { ...styles.topBarItemSelected }, ...styles.topBarItem }}>
+                    <span style={styles.topBarItemText}>
+                      {item.label}
                     </span>
                   </div>
+                </div>;
+              })
+              }
+            </div>
+            <div style={styles.innerContainer} id='childView'>
+              {selectionId === 1 &&
+                <GenomicOverviewComponent
+                  participantCount={participantCount}
+                  chartData={chartData}
+                />}
+              {selectionId === 2 &&
+                <GenomicSearchComponent
+                  onSearchInput={(searchWord: string) => {
+                    this.handleSearchTerm(searchWord);
+                    this.setState({ searchTerm: searchWord });
+                  }}
+                  onPageChange={(info) => { this.handlePageChange(info); }}
+                  onRowCountChange={(info) => { this.handleRowCountChange(info); }}
+                  onSortClick={(sortMetadata) => { this.handleSortClick(sortMetadata); }}
+                  onFilterSubmit={(filteredMetadata: GenomicFilters) => {
+                    this.handleFilterSubmit(filteredMetadata);
+                  }}
+                  currentPage={currentPage}
+                  rowCount={rowCount}
+                  variantListSize={variantListSize}
+                  loadingVariantListSize={loadingVariantListSize}
+                  loadingResults={loadingResults}
+                  searchResults={searchResults}
+                  participantCount={participantCount}
+                  searchTerm={searchTerm}
+                  filterMetadata={filterMetadata} />}
+
+              {selectionId === 3 &&
+                <GenomicFaqComponent closed={() => this.handleFaqClose()} />}
+              <div style={styles.faqHeading}>
+                <div className='faq-heading-text'>Questions about genomics?
+                  <span style={styles.faqLink} onClick={() => this.topBarClick(3)}>Learn More</span>
                 </div>
               </div>
             </div>
           </div>
-        </React.Fragment>
-      );
+          <div style={styles.innerContainer} id="childView">
+            {selectionId === 1 && (
```

Comment:
Is this duplicated ?

Reply:
Strange I thought I pushed this fix. 
```suggestion
            {selectionId === 1 && (
```

Reply:
i dont even know how I made a 'suggested change' I will fix it